addons/sourcemod/scripting/AdminLogging.sp

```diff
@@ -91,6 +101,18 @@ public Action OnLogAction(Handle source, Identity ident, int client, int target,
 	if (adminID == INVALID_ADMIN_ID && client > 0)
 		return Plugin_Continue;
 
+	char sEscapedMessage[WEBHOOK_MSG_MAX_SIZE];
+	FormatEx(sEscapedMessage, sizeof(sEscapedMessage), "%s", message);
+	ReplaceString(sEscapedMessage, sizeof(sEscapedMessage), "`", "\\`");
```

Comment:
maybe put this in a common include file to avoid duplicating this code everywhere

Reply:
I've already thought about this issue.
Depending on the plugin and the content of the string, the string replacement can vary.
I concluded that since `ReplaceString` is a native function of SM, it's not a significant concern.